PostgreSQL upgrade from 9.3 to 13.4 (#2216)
chef/supermarket: dev-docs/pg-post-upgrade-steps.md

> ### Follow these steps to upgrade a PostgreSQL major version

Reviewer: Since this upgrade changes attributes and requires one time manual processes before the upgrade we need to consider it to be a major version upgrade. We should make this upgrade the 5.0 release so we properly communicate the work involved to our customers.

Reviewer 2: We can remove this file as we have included all the steps in the `supermarket_upgrade.md` file. Will remove this file.

> ### Follow these steps in case Postgres major version is being upgraded.

Reviewer 3: ```suggestion
### Follow these steps to upgrade a PostgreSQL major version
```

>  - Stop currently running supermarket application

Reviewer 3: ```suggestion
 - Stop the supermarket application
```
